combat-trainer.lic

```diff
@@ -333,6 +333,15 @@ class LootProcess
     echo("  @last_rites: #{@last_rites}") if $debug_mode_ct
     @last_rites_timer = Time.now - 600
 
+    if settings.box_loot_limit
+      @stop_hunt_on_box_limit = settings.stop_hunt_on_box_limit
+      @box_nouns = get_data('items').box_nouns
+      @box_loot_limit = settings.box_loot_limit
+      @current_box_count = get_boxes(settings.picking_box_source).size + get_boxes(settings.picking_pet_box_source).size
+      echo("  @current_box_count: #{@current_box_count}")
```

Comment:
Please also echo `stop_hunt_on_box_limit`

Reply:
ah, thats for the stopping functionality. removing it.